lemur/plugins/lemur_aws/plugin.py

```diff
@@ -112,9 +112,9 @@ def get_elb_endpoints(account_number, region, elb_dict):
             dnsname=elb_dict["DNSName"],
             type="elb",
             port=listener["Listener"]["LoadBalancerPort"],
-            certificate_name=iam.get_name_from_arn(
-                listener["Listener"]["SSLCertificateId"]
-            ),
+            certificate_name=iam.get_name_from_arn(listener["Listener"]["SSLCertificateId"]),
+            certificate_path=iam.get_path_from_arn(listener["Listener"]["SSLCertificateId"]),
```

Comment:
persisting the new information in the returned dict, and later into the endpoint table